src/main/java/randoop/reflection/RawSignature.java

```diff
@@ -34,7 +34,7 @@
   /**
    * Create a {@link RawSignature} object with the name and parameterTypes.
    *
-   * @param packageName the package name of the class; null for the unnamed package
+   * @param packageName the package name of the class
```

Comment:
I would still document what is used for the unnamed package (even if it differs in Java 8 and 9).